hparams.py

```diff
@@ -1,9 +1,9 @@
-import tensorflow as tf
+import hparam_tf.hparam
```

Comment:
Could this be simply `import hparam_tf`?

Reply:
Just get the file hparam.py out of the subfolder, rename it to hparam_tf.py and import it directly as you suggest. I think it will work.
I just created the subfolder to add the readme where its origin is explained.

Reply:
Ah, I see. I misunderstood there's ``__init__py`` in the `hparams_tf` directory.